src/fragments/lib/graphqlapi/graphql-from-node.mdx

```diff
@@ -56,6 +56,7 @@ The examples on this page use [`node-fetch`](https://www.npmjs.com/package/node-
   }
 }
 ```
+NOTE: The examples bellow are using ES module syntax and currently not supported by amplify mock function. If you wish to use amplify mock please use commonjs syntax. 
```

Comment:
Hey @mstoyanovv :wave: thanks for filing this! Would you be open to providing a short CJS example? Or adding the callout to use `node-fetch` v2 with commonjs? 

IAM example:

```js
const crypto = require('@aws-crypto/sha256-js');
const { defaultProvider } = require('@aws-sdk/credential-provider-node');
const { SignatureV4 } = require('@aws-sdk/signature-v4');
const { HttpRequest } = require('@aws-sdk/protocol-http');
const fetch = require('node-fetch');
const { Request } = fetch

const { Sha256 } = crypto;
const GRAPHQL_ENDPOINT = process.env.API_<YOUR_API_NAME>_GRAPHQLAPIENDPOINTOUTPUT;
const AWS_REGION = process.env.AWS_REGION || 'us-east-1';

const query = /* GraphQL */ `
  query LIST_TODOS {
    listTodos {
      items {
        id
        name
        description
      }
    }
  }
`;

/**
 * @type {import('@types/aws-lambda').APIGatewayProxyHandler}
 */
exports.handler = async (event) => {
  console.log(`EVENT: ${JSON.stringify(event)}`);

  const endpoint = new URL(GRAPHQL_ENDPOINT);

  const signer = new SignatureV4({
    credentials: defaultProvider(),
    region: AWS_REGION,
    service: 'appsync',
    sha256: Sha256
  });

  const requestToBeSigned = new HttpRequest({
    method: 'POST',
    headers: {
      'Content-Type': 'application/json',
      host: endpoint.host
    },
    hostname: endpoint.host,
    body: JSON.stringify({ query }),
    path: endpoint.pathname
  });

  const signed = await signer.sign(requestToBeSigned);
  const request = new Request(endpoint, signed);

  let statusCode = 200;
  let body;
  let response;

  try {
    response = await fetch(request);
    body = await response.json();
    if (body.errors) statusCode = 400;
  } catch (error) {
    statusCode = 500;
    body = {
      errors: [
        {
          message: error.message
        }
      ]
    };
  }

  return {
    statusCode,
    body: JSON.stringify(body)
  };
};
```

Reply:
@josefaidt I have added the callout for node-fetch v2. Did not include snippet as I think it might confuse the user. Once amplify mock supports es modules only the 'note' part will have to be removed from the documentation.

Reply:
hey @josefaidt  just a reminder to check the pr :)

Reply:
Hey @mstoyanovv so sorry for the delay here! I've since added a commonjs example to the same docs page with a callout to node-fetch v2, but I do appreciate how you've taken the step to include the appropriate Content-Type header. Would you mind merging the latest from main?